frontend/src/layouts/BaseLayout.vue

```diff
@@ -76,10 +82,17 @@
             </connect-wallet>
           </div>
         </div>
-        <!-- Alpha warning -->
-        <div class="dark-toggle text-center text-negative text-bold q-my-md">
-          This is beta software. Use at your own risk.
-          <router-link class="hyperlink" :to="{ path: '/faq#security' }">Learn more</router-link>
+        <!-- Legacy warning -->
+        <div
+          v-if="!isAccountSetup && isAccountSetupLegacy"
+          class="dark-toggle text-center text-bold q-my-md q-pa-md"
+          style="border-radius: 15px"
+          :style="isDark ? 'color: #FFFAEB; background-color: #7c5e10' : 'color: #513C06; background-color: #fcefc7'"
+        >
+          We've upgraded our name resolution system. Use the
+          <router-link class="hyperlink" :to="{ name: 'setup' }">Setup</router-link> page to submit one transaction
+          which migrates you to the updated system. You won't be able to receive Umbra transactions until you've done
+          so. <router-link class="hyperlink" :to="{ path: '/TODO' }">Learn more</router-link>.
```

Comment:
Copy for the new question:

"Why do I need to setup my account again?"

If you configured your Umbra account while the service was still in Beta— before October 2021— you need to redo setup. This requires one signature and one transaction.

This additional setup step is due to an upgrade to Umbra's stealth key registry system. The old system relied on records placed on the resolver of the ENS record associated with a user's address. The new system is much simpler, and uses a simple mapping between the user's address and their stealth keys.

The new system was developed in response to user feedback during Umbra's Beta period. The old system required multiple transactions, and also required senders use the receiver's ENS name, which caused confusion. The new system requires only a single transaction, and senders can provide either the receiver's ENS name or their regular Ethereum address. Overall, the new system is simpler both to configure and use.

For an even more in depth and technical explanation of this upgrade, see [this issue](https://github.com/ScopeLift/umbra-protocol/issues/214) on Umbra's Github repository.

Reply:
Added in 1b76a6082a9698be7b2d4695c301c5c899509aa4